src/main/java/org/folio/dew/service/UserPermissionsService.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.dew.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dew.client.EurekaUserPermissionsClient;
+import org.folio.dew.client.OkapiUserPermissionsClient;
+import org.folio.spring.FolioExecutionContext;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+@RequiredArgsConstructor
+@Log4j2
+@Service
+public class UserPermissionsService {
+
+  public static final String BULK_EDIT_INVENTORY_VIEW_PERMISSION = "bulk-operations.item.inventory.get";
```

Comment:
Saving permissions in the enum looks more natural.

Reply:
Updated